docs/userGuide/contentAuthoring.md

```diff
@@ -300,19 +305,20 @@ If you need additional footers for different pages, you can create footer files
 </footer>
 ```
 
-After which, you must specify the file within [front matter](#front-matter)'s `footer` attribute to render the footer.
+After which, you must specify the file within a page's [front matter](#front-matter)'s `footer` attribute to render the footer.
 
 ```html
-<!-- In the file that you want to include a footer -->
+<!-- In the page that you want to include a footer -->
 <frontmatter>
   footer: myFooterFile.md
 </frontmatter>
 ```
 
 Note:
 - Only one footer file can be specified in the [front matter](#front-matter) per page, and you must include its file extension.
+- Updating the footer file requires a restart of the server to reflect the changes on [live preview](userQuickStart.html#preview-your-site).

```

Comment:
If you are using [live preview](userQuickStart.html#preview-your-site), you must restart the server when updating the footer file to see the changes.